recipes/xrandr.lua

```diff
@@ -0,0 +1,146 @@
+--- Separating Multiple Monitor functions as a separeted module (taken from awesome wiki)
+
+local awful     = require("awful")
+local naughty   = require("naughty")
+local beautiful = require("beautiful");
+
+-- A path to a fancy icon
+local icon_path = "";
+
+-- Get active outputs
+local function outputs()
+   local outputs = {}
+   local xrandr = io.popen("xrandr -q")
+
+   if xrandr then
+      for line in xrandr:lines() do
+         local output = line:match("^([%w-]+) connected ")
+         if output then
+            outputs[#outputs + 1] = output
+                                   end
+      end
+      xrandr:close()
```

Comment:
Well, ok. We do not yet provide a way to query the names of disables outputs. We could just add an API for that to awesome. Dunno if that's a good idea.